count_test.go

```diff
@@ -0,0 +1,34 @@
+package storm
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCount(t *testing.T) {
+	dir, _ := ioutil.TempDir(os.TempDir(), "storm")
+	defer os.RemoveAll(dir)
+	db, _ := Open(filepath.Join(dir, "storm.db"))
+
+	for i := 0; i < 100; i++ {
+		w := User{Name: "John", ID: i + 1, Slug: fmt.Sprintf("John%d", i+1), DateOfBirth: time.Now().Add(-time.Duration(i*10) * time.Minute)}
+		err := db.Save(&w)
+		assert.NoError(t, err)
+	}
+
+	count, _ := db.Count(User{})
+	assert.Equal(t, 100, count)
+
+	w := User{Name: "John", ID: 101, Slug: fmt.Sprintf("John%d", 101), DateOfBirth: time.Now().Add(-time.Duration(101*10) * time.Minute)}
+	err := db.Save(&w)
+	assert.NoError(t, err)
+
+	count, _ = db.Count(User{})
+	assert.Equal(t, 101, count)
```

Comment:
There is count on:

1) Node
2) DB
3) When in a transaction and not

Your tests cover only the first of the above.